startup/dispatch/project.py

```diff
@@ -0,0 +1,80 @@
+##########################################################################
+#
+#  Copyright (c) 2018, Image Engine Design Inc. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of John Haddon nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+## \todo : This entire file was copied directly from startup/gui/project.py
```

Comment:
Having the same settings seems like a good idea, at least until we get some experience with the new app in practice. What I did for `startup/view/ocio.py` was to just symlink it to `startup/gui/ocio.py`. I'm not sure that's great, but I think it's preferable to a copy that'll inevitably get out of sync.

Reply:
I should clarify, it’s not an exact copy. All the code here does exist in the gui file, but that one has other settings we don’t need here (bookmarks, image format) and some that we explicitly want the opposite of (local executeInBackground userDefault).

What I would like to be consistent across all apps is the registered dispatchers and the location of their job directories.

To do that, I’d need the project variables on the script to be consistent (and exist) as well, or to move the default job directories somewhere else inside ~/gaffer

Reply:
I guess I made this comment after you signed off, or it got buried in the slew of other comments. What are your thoughts now you know its not an exact copy of the gui file?

Reply:
I'm still inclined to just symlink. It seems like the bookmarks and image format should be harmless and might even be useful (it does look like the bookmarks want a ScriptWindow, but that could be fixed). The "executeInBackground" default doesn't seem project-related at all really, so we could move it somewhere else in the gui startup, maybe "startup/gui/localDispatcher.py"?

Reply:
Still todo. Though that symlink suggestion seems quite fragile now I think about it... any new setting accicentally put in "startup/gui/proect.py" would affect the dispatch app, possibly without the committer realizing it.